content/k8s/manual-app-deployment/ec2-setup/index.md

```diff
@@ -0,0 +1,30 @@
+---
```

Comment:
Since we will automate the instance creation process, we can modify this page.

Instead of "Setup EC2", we can a have page named "Accessing your EC2 instance", where we tell them how to:
- Save the `key.pem` file with the content given by us
- Change the `key.pem` file permissions (if needed)
- Use SSH to access their instance

We also need to make this OS agnostic, maybe they have Windows/Linux/OSX.

What do you think @t-witchy?